jrodriguezdiazz/book-search-engine
# Add card component for books (BookCard)

## Description

- Add card component for books (`BookCard`)
- Add a template book grid (`GridBooksCard`)
- Update the options that appear in the search in the search engine 
- Update the Google Books API endpoint 
- Add dependency: [`html-react-parser`](https://www.npmjs.com/package/html-react-parser)



---

## Motivation and Context





This feature solves the problem of displaying the list of books when using the search engine

---

## How Has This Been Tested?

1. Visit the next page: https://book-search-engine-25c3r2h7o-jrodriguezdiazz.vercel.app/
2. You should see a grid with 10 books with title, image, description and other information
3. Click in the search input
4. You should see a list of books name (and not a list of films name )





---

## Screenshots (if appropriate):

| Desktop | Mobile |
| ------- | ------ |
| ![image](https://user-images.githubusercontent.com/24766287/144951644-463cde2c-b917-47da-860f-7f5467b1a411.png) | ![image](https://user-images.githubusercontent.com/24766287/144951723-e073757c-e0c7-414a-9aab-39e4629e073e.png) |

---

## Types of Changes




---

## Checklist:





---

# Other Notes
